utils/index.ts

```diff
@@ -184,3 +184,32 @@ export function convertChannelIdToLikerId (channelId: string) {
 export function convertLikerIdToChannelId (likerId: string) {
   return `@${likerId}`
 }
+
+export function copyToClipboard (text: string): void {
+  const toast = useToast()
+  const copyText = document.createElement('p')
+  copyText.textContent = text
+  document.body.appendChild(copyText)
+
+  const selection = window.getSelection()
+  const range = document.createRange()
+
+  if (!selection) {
+    return
+  }
+
+  range.selectNode(copyText)
+  selection.removeAllRanges()
+  selection.addRange(range)
+
+  document.execCommand('copy')
+  selection.removeAllRanges()
+
+  document.body.removeChild(copyText)
```

Comment:
Should try `navigator.clipboard` first
`https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText`

Reply:
[🧑‍💻 Use navigator.clipboard with fallback](https://github.com/likecoin/nft-book-press/pull/215/commits/b4dca9001dc353b57c9ef5ae71a894d5e06867ab)